src/assets/styles/system.scss

```diff
@@ -181,9 +185,9 @@ $line-height-heading: 1.5;
 // @param  {Color} $color of background
 // @return {Boolean}
 @function color-difference($foreground, $background) {
-  $r: (max(red($foreground), red($background))) - (min(red($foreground), red($background)));
-  $g: (max(green($foreground), green($background))) - (min(green($foreground), green($background)));
-  $b: (max(blue($foreground), blue($background))) - (min(blue($foreground), blue($background)));
+  $r: (max(color.channel($foreground, "red", $space: rgb), color.channel($background, "red", $space: rgb))) - (min(color.channel($foreground, "red", $space: rgb), color.channel($background, "red", $space: rgb)));
```

Comment:
When I run `npm run styleguide`, I get the following error:

```
Undefined function.
    ╷
188 │   $r: (max(color.channel($foreground, "red", $space: rgb), color.channel($background, "red", $space: rgb))) - (min(color.channel($foreground, "red", $space: rgb), color.channel($background, "red", $space: rgb)));
    │            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ╵
  src/assets/styles/system.scss 188:12           color-difference()
  src/assets/styles/system.scss 210:7            set-text-color()
  src/components/LuxAutocompleteInput.vue 40:12  root stylesheet
```

Reply:
Ahhhh never mind!  I forgot to `rm -rf node_modules` and `npm install`.  This is actually working, please disregard!